packages/graphql-tag-pluck/src/visitor.ts

```diff
@@ -142,12 +142,53 @@ export type PluckedContent = {
   };
 };
 
+function defaultPluckStringFromFile(
+  code: string,
+  { start, end }: { start: number; end: number },
+  options: { skipIndent?: boolean } = {},
+) {
+  return freeText(
+    code
+      // Slice quotes
+      .slice(start + 1, end - 1)
+      // Erase string interpolations as we gonna export everything as a single
+      // string anyway
+      .replace(/\$\{[^}]*\}/g, '')
+      .split('\\`')
+      .join('`'),
+    options.skipIndent,
+  );
+}
+
+function defaultIsGqlTemplateLiteral(node: any, options: { gqlMagicComment?: string }) {
+  const leadingComments = node.leadingComments;
+
+  if (!leadingComments) {
+    return;
+  }
+  if (!leadingComments.length) {
+    return;
+  }
+
+  const leadingComment = leadingComments[leadingComments.length - 1];
+  const leadingCommentValue = leadingComment.value.trim().toLowerCase();
+
+  if (leadingCommentValue === options.gqlMagicComment) {
+    return true;
+  }
+
+  return false;
+}
```

Comment:
These are just extracted from the code below so that they can be overwritten by params.